recipes/llvmlite/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "llvmlite" %}
+{% set version = "0.20.0" %}
+{% set sha256 = "b2f174848df16bb9195a07fec102110a06d018da736bd9b3570a54d44c797c29" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+    
+build:
+  skip: True  # [win and py2k]
+  number: 0
+  script: python setup.py install  --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - enum34  # [py2k]
+    - llvmdev
+  run:
+    - python
+    - enum34  # [py2k]
+
+test:
+  imports:
+    - llvmlite
+    - llvmlite.binding
+    - llvmlite.ir
+    - llvmlite.llvmpy
+    - llvmlite.tests
```

Comment:
Is there some sort of test that we can do that is a little more advanced than importing? Maybe there is a simple Python example in the README or the docs that we can borrow? This would be best placed either in `commands` (if it is very simple) or in a `run_test.py`.

Reply:
Should we make use of the tests wriiten for ``llvmlite``- https://github.com/numba/llvmlite/blob/master/llvmlite/tests/test_valuerepr.py?

Reply:
Sure reusing existing tests is a good idea. We don't need to run all of them as that may be too slow, but selected tests would be a good indicator that this works ok.